src/components/QuickOpenModal.js

```diff
@@ -233,10 +233,10 @@ export class QuickOpenModal extends Component<Props, State> {
   traverseResults = (e: SyntheticKeyboardEvent<HTMLElement>) => {
     const direction = e.key === "ArrowUp" ? -1 : 1;
     const { selectedIndex, results } = this.state;
-    const resultCount = this.getResultCount();
+    const resultCount =
+      this.getResultCount() > 100 ? 100 : this.getResultCount();
```

Comment:
this looks like we might need to improve Result Count. 

Also, we might want `maxResults` as a variable